src/generator/operations.ts

```diff
@@ -412,9 +412,13 @@ function createProtocolRequest(client: string, op: Operation, imports: ImportMan
     let setOptionsPrefix = !bodyParam!.required;
     // default to the body param name
     let body = bodyParam!.language.go!.name;
+    let skipBody = false;
     if (bodyParam!.schema.type === SchemaType.Constant) {
       // if the value is constant, embed it directly
       body = formatConstantValue(<ConstantSchema>bodyParam!.schema);
+      if ((<ConstantSchema>bodyParam!.schema).valueType.type === SchemaType.Boolean) {
```

Comment:
Why do we skip this only for bools?  Probably good to add a comment here.

Reply:
Added the comment, also added an issue to improve this check: #348